src/Form/ForgotPasswordForm.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\EmailType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Validator\Constraints\Email;
+use Symfony\Component\Validator\Constraints\NotBlank;
+
+final class ForgotPasswordForm extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('email', EmailType::class, [
+                'attr' => ['autocomplete' => 'email'],
+                'constraints' => [
+                    new NotBlank(
+                        message: 'Please enter your email',
+                    ),
+                    new Email(
+                        message: 'Please enter a valid email address',
+                    ),
+                ],
+                'mapped' => false,
```

Comment:
this form isn't backed by a class, so is this needed?

Reply:
good call - I copied these classes from the maker files.